### DISCUSSION TITLE
Sound->output combined with mike input to > blackhole audio driver?

### DISCUSSION THREAD

**Discussion by vlad-cotfas** [ExistentialAudio/BlackHole#598] [Q&A]
I`m trying to achieve the combination of the sound output (regardless of "external headphones" or "macbook pro speakers") combined with microphone (regardless of external mike or mac mike) into blackhole driver as output. 

Or at least routing the sound output as well to blackhole output driver (like dividing the output sound, to be available also on sound/speakers but also on the blackhole output)

Is this possible with BlackHole driver? And what kind of configuration I have to do?
Thank you!

**Comment by devinroth** [comment]
There’s a number of guides at https://existential.audio/blackhole/support

**Comment by vlad-cotfas** [comment]
Hello Devin,
Thank you for your reply. I`ve looked a little bit into https://existential.audio/blackhole/support/
I still don't understand, I can achieve with:
-Multi-output device? -this option will not help me
-Aggregate device? -I can achieve with aggregate? 
(it is also okay if I only split the audio out to blackhole driver output without microphone)

*Basically blackhole is only an audio device in system settings and with "Audio MIDI Setup" I can do the magic? 

EDIT: One note is that I do not want to change the output driver in the System app so I do not loose the volume 

Kind regards,
Vlad

**Comment by vlad-cotfas** [comment]
To be easier understood I made a diagram 😁:

![Sound Diagram](https://user-images.githubusercontent.com/3340367/145702786-8e758008-ac00-4d2b-acaa-ea64a0c73a97.jpg)

**Comment by vlad-cotfas** [comment]
Any news?

**Comment by vlad-cotfas** [comment]
Any news?

**Comment by vlad-cotfas** [comment]
Any news?

**Comment by vlad-cotfas** [comment]
Hey @devinroth what do you think? Can I be able to implement this ?

Or I should use loopback?

**Reply by saltorico** [reply]
did you ever get it to work like this? I'm trying to mix my microphone and the macos Sonoma transcriptions text-to-speech. In one of the gazillion options using "backgroundmusic" i got a test call in Teams to work and actually let me hear the output.

**Comment by devinroth** [comment]
With all due respect I don’t have the time to walk everyone through every possible combination how how to set things up.

Feel free to go with something else if you want to pay for support.

Devin
